src/lib/components/Navbar.svelte

```diff
@@ -173,7 +184,7 @@ The navbar component.
             </a>
         {/if}
         <!-- Text Appearance Options Menu -->
-        {#if $globalConfig.mainFeatures['text-font-size-slider'] || $globalConfig.mainFeatures['text-line-height-slider']}
+        {#if $globalConfig.mainFeatures['text-font-size-slider'] || $globalConfig.mainFeatures['text-line-height-slider'] || countThemesNormal || countThemesDark || countThemesSepia}
             <Dropdown>
                 <svelte:fragment slot="label">
                     <TextAppearanceIcon />
```

Comment:
I added the above on lines 53-61 because it made more sense to me to have these variables and do the comparison that way, if you want me to changed it to the count way you said I can.